yb-voyager/src/dbzm/dbzm.go

```diff
@@ -82,10 +107,22 @@ func (d *Debezium) GetExportStatus() (*ExportStatus, error) {
 func (d *Debezium) Stop() error {
 	if d.IsRunning() {
 		log.Infof("Stopping debezium...")
-		err := d.cmd.Process.Kill()
+		err := d.cmd.Process.Signal(syscall.SIGTERM)
+		go func() {
+			// wait for a certain time for debezium to shut down before force killing the process.
+			sigtermTimeout := 100
+			time.Sleep(time.Duration(sigtermTimeout) * time.Second)
+			if d.IsRunning() {
+				log.Warnf("Waited %d seconds for debezium process to stop. Force killing it now.", sigtermTimeout)
+				d.cmd.Process.Kill()
```

Comment:
handle the `error` returned by `Kill()` function.
You might need to override the `err` variable at line 110

Reply:
Good catch! Handled it. I am basically os.Exiting here if there is an error. Don't see an alternative.